models/aliyun_ftasr_api_zh/read_oss_file.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# -*- coding: utf8 -*-
+'''
+录音文件识别 接口说明
+https://help.aliyun.com/document_detail/90727.html
+通过ossutil命令获取文件URL
+egs: 为目标存储空间examplebucket下的文件exampleobject.png生成文件URL，并指定超时时间为3600秒。
+     ./ossutil64 sign oss://examplebucket/exampleobject.png --timeout 3600
+'''
+import os
+import sys
+import codecs
+
+if __name__ == "__main__":
+    if len(sys.argv) != 3:
+        sys.stderr.write("read_oss_file.py <wav_scp> <oss_out_scp>\n")
+        exit(-1)
+
+    idxList = []
+    wavFile = codecs.open(sys.argv[1], 'r',  'utf8') 
+    # wav OSS存放地址
+    ossPath = "oss://example/wav/"
```

Comment:
I'm kinda confused with this `ossPath`:
1. who own this bucket? is this bucket accessible from public? Some of SpeechIO test sets are not open-sourced yet, uploading files to a public oss path will leak those test sets.
2. even if this is an oss path filled by leaderboard caller, does it mean that leaderboard caller need to create this bucket and directory manually from Aliyun web-console? Things like these steps can not be automated within an automate pipeline.

Reply:
Also, how is this OSS bucket charged? Does this mean the leaderboard caller need to maintain this bucket and pay for uploading plus 7x24 storage fees?